packages/matchbox/src/components/Tooltip/Tooltip.js

```diff
@@ -1,124 +1,166 @@
+    const contentStyles = {
+      ...systemProps,
+      ...(dark ? { bg: 'gray.1000' } : {}), // TODO deprecate in favor of system props
+    };
 
     return (
-      <span className={wrapperClasses}>
-        <span className={styles.Tooltip} style={{ ...offset, width }}>
-          <span className={styles.Tip} />
-          <div className={styles.Content}>{content}</div>
-        </span>
-      </span>
+      <StyledTooltipContainer
+        display="block"
+        position="absolute"
+        top={positionTop ? 'auto' : '100%'}
+        bottom={positionTop ? '100%' : 'auto'}
+        left={positionLeft ? 'auto' : '0'}
+        right={positionLeft ? '0' : 'auto'}
+        // Tooltip gap to activator
+        mt={positionTop ? '0' : '100'}
+        mb={positionTop ? '100' : '0'}
+        visible={!disabled && hover}
+      >
+        <StyledContent
+          position="relative"
+          fontSize="200"
+          lineHeight="200"
+          p="300"
+          borderRadius="100"
+          bg="gray.1000"
+          color="white"
+          textAlign="left"
+          width={width}
+          zIndex="1"
+          {...contentStyles}
+        >
+          <Box overflow="hidden" maxHeight="11rem">
+            {content}
+          </Box>
+        </StyledContent>
+      </StyledTooltipContainer>
     );
   }
 
-  renderActivator = ({ activatorRef }) => (
-    <span
-      className={styles.Activator}
-      onMouseOver={this.handleMouseOver}
-      onMouseOut={this.handleMouseOut}
-      ref={activatorRef}>
-      {this.props.children}
-    </span>
-  )
-
-  render() {
+  function renderActivator({ activatorRef }) {
     return (
-      <TooltipOverlay
-        eventDebounce={this.props.eventDebounce}
-        portalId={this.props.portalId}
-        renderTooltip={this.renderTooltip}
-        renderActivator={this.renderActivator} />
+      <Box
+        display="inline-block"
+        onMouseOver={handleMouseOver}
+        onMouseOut={handleMouseOut}
+        ref={activatorRef}
+      >
+        {props.children}
+      </Box>
     );
   }
+
+  return (
+    <TooltipOverlay
+      eventDebounce={props.eventDebounce}
+      portalId={props.portalId}
+      renderTooltip={renderTooltip}
+      renderActivator={renderActivator}
+    />
+  );
 }
+Tooltip.displayName = 'Tooltip';
+Tooltip.propTypes = {
+  content: PropTypes.node,
+  /**
+   * Disables hover events
+   */
+  disabled: PropTypes.bool,
+  dark: deprecate(PropTypes.bool, 'Use system props to set styles'),
+  active: PropTypes.bool,
+  left: PropTypes.bool,
+  right: PropTypes.bool,
+  top: PropTypes.bool,
+  bottom: PropTypes.bool,
+  horizontalOffset: PropTypes.string, // TODO Either deprecate or style Tooltip tips
+  /**
+   * Disables automatic positioning
+   */
+  forcePosition: PropTypes.bool,
+  children: PropTypes.node,
+  /**
+   * These props are provided automatically through the Overlay component
+   */
+  preferredDirection: PropTypes.shape({
```

Comment:
follow-up: Shouldn't this be an array?

Reply:
Potentially yeah, also these props don't really need to be defined here and only `top` and `left` are used. Can revise this a bit when we make API updates